src/point_one/fusion_engine/messages/measurements.h

```diff
@@ -809,10 +809,8 @@ struct P1_ALIGNAS(4) RawVehicleSpeedOutput : public MessagePayload {
  * @ingroup measurement_messages
  *
  * This message is an input to the device, used to convey the wheel encoder tick
- * counts for one or more wheels, received through software (e.g., vehicle CAN
- * bus) or captured in hardware from external voltage pulses. The number and
- * type of wheels expected, and the interpretation of the tick count values,
- * varies by vehicle.
+ * counts for one or more wheels. The number and type of wheels expected, and
+ * the interpretation of the tick count values, varies by vehicle.
```

Comment:
So I think we discussed that we should make it clear this is a software-only message, as in the ticks are extracted from somewhere else and they're sent via these messages.

Reply:
I think that should be clear after removing the other part? As it says, this message is an input to the device, not an output from the device. The only way for this message to exist is for an external processor to generate it in software.

Reply:
Ah I see - you're right, looks good to me.